app/main.py

```diff
@@ -1 +1,16 @@
-# write your code here
+from app.players.elves.elf import Elf
+from app.players.dwarves.dwarf import Dwarf
+
+
+def calculate_team_total_rating(team_list: list) -> int:
```

Comment:
Specify inner type of list
```suggestion
def calculate_team_total_rating(team_list: list[Player]) -> int:
```